Jenkinsfile

```diff
@@ -6,3 +6,11 @@ common {
   // change across version branches.
   upstreamProjects = 'kafka-trunk'
 }
+
+docker_oraclejdk8 {
+    dockerRegistry = '368821881613.dkr.ecr.us-west-2.amazonaws.com/'
+    dockerRepos = ['confluentinc/cp-base']
+    slackChannel = '#tools-eng'
+    upstreamProjects = ['confluentinc/confluent-docker-utils']
+    withPush = true
```

Comment:
- change withPush to dockerPush (withPush was deprecated)
- use 'clients-eng' for docker_oraclejdk8's slackChannel
- add 'kafka-trunk' to docker_oraclejdk8's upstreamProjects, preserving the old comment from common{}.
- add nodeLabel = 'docker-oraclejdk7' to the docker_oraclejdk8 block
- remove the common{} block